### ISSUE TITLE
Can't build on Ubuntu 16.04.2 LTS

### ISSUE THREAD

**Issue by Frantzd** [qtlmovie/qtlmovie#8]
Hello,

I've tried to build QtlMovie-1.12 with build/build-deb.sh and at the end 4ive got this error :

(...)
make[1]: La cible « first » n'a pas été refabriquée à cause d'erreurs.
Makefile:176 : la recette pour la cible « sub-QtlMovie-make_first-ordered » a échouée
make: *** [sub-QtlMovie-make_first-ordered] Erreur 2
make: La cible « first » n'a pas été refabriquée à cause d'erreurs.
install: impossible d'évaluer '/home/frantz/temp/QtlMovie-1.12/build-QtlMovie-Desktop-Release/QtlMovie/QtlMovie': Aucun fichier ou dossier de ce type
strip: « /tmp/qtlmovie-20942/usr/bin/QtlMovie »: pas de tel fichier
dpkg-deb : construction du paquet « qtlmovie » dans « /home/frantz/temp/QtlMovie-1.12/installers/qtlmovie_1.12_amd64.deb ».

Regards.

**Comment by lelegard** [comment]
Hello,

What are the error messages before make failed? Could you please attach a complete log file of the build?

Before building QtlMovie, did you install the pre-requisites as described in the [build instructions](http://qtlmovie.sourceforge.net/doc/qtlmovie-build.html), namely:

```
apt-get install qtcreator qtbase5-dev qt5-default qtmultimedia5-dev
```

**Comment by Frantzd** [comment]
Hello,

Yes I did install the  pre-requisites. I attach a complete log file.

Regards
[install_log.txt](https://github.com/qtlmovie/qtlmovie/files/970012/install_log.txt)

**Comment by lelegard** [comment]
Hello,

QtlMovie builds correctly on Ubuntu 16.10 and 17.04. On Ubuntu 16.04, the compilation fails due to the old version of Qt: version 5.5 while the latest one is 5.8. Older versions of QtlMovie used to compile with Qt 5.5 but recent changes used a feature from Qt 5.6.

If have patched the source code so that QtlMovie compiles correctly with Qt 5.5 on Ubuntu 16.04.

The changes will be integrated in the next version of QtlMovie. In the meantime, to compile QtlMovie on Ubuntu 16.04, please get the latest fix from the git repository.

Either from GitHub:
```
git clone https://github.com/qtlmovie/qtlmovie.git qtlmovie
```
Or from SourceForge:
```
git clone https://git.code.sf.net/p/qtlmovie/code qtlmovie
```

Sorry for the inconvenience.

Closing this issue, fixed in commit 7bc4c976cbe7b5735d38dd52e680d3c17557dd6a

**Comment by Frantzd** [comment]
Thank's a lot it works well :-)

Regards
